src/features/MentorPage/MentorChips/MentorChips.tsx

```diff
@@ -0,0 +1,56 @@
+import { ChipProps } from '../../../components/Chip/types';
+import styled from 'styled-components';
+import Chip from '../../../components/Chip';
+import ShowMoreChips from './ShowMoreChips';
+
+/**
+ * Logic to show skill chips. Array of SkillPillItems
+ * should be passed to SkillPill, where styles and layout
+ * will be added. Should call MentorCards when chips are
+ * selected?
+ */
+
+const MentorSkillChips = ({ items }: { items: Array<ChipProps> }) => {
+  return (
+    <ChipContainer>
+      <SkillChips id="skill-chips">
+        {items.map(item => (
```

Comment:
If you want the items to be in alphabetical order, you have to sort them before mapping them. ( Issue mentioned in yesterday meeting )

Reply:
Ah yeah, I totally forgot to implement this